app/uk/gov/hmrc/tradergoodsprofiles/controllers/RemoveRecordController.scala

```diff
@@ -57,19 +58,18 @@ class RemoveRecordController @Inject() (
   )(implicit ec: ExecutionContext): EitherT[Future, Result, RemoveRecordRequest] =
     EitherT.fromEither(
       request.body.validate[RemoveRecordRequest].asEither.leftMap { errors =>
-        InvalidActorIdErrorResponse(
-          dateTimeService.timestamp,
-          "Missing or invalid mandatory request parameter"
-        ).toResult
+        InvalidErrorResponse(uuidService.uuid, InvalidRequestParameter, InvalidActorMessage, InvalidActorId).toResult
       }
     )
 
   def validateRecordId(recordId: String): EitherT[Future, Result, String] =
     EitherT.fromEither[Future](
       Try(UUID.fromString(recordId).toString).toEither.left.map(_ =>
-        InvalidRecordIdErrorResponse(
-          dateTimeService.timestamp,
-          "Invalid record ID supplied for eori number provided"
+        InvalidErrorResponse(
+          uuidService.uuid,
```

Comment:
correlationId ID here has not meaning and it will only create confusion for the user or user support team as this is used to track the request downstream (to EIS), but in here we haven\t set the request to EIS yet